tests/Unit/IO/Test_VolumeData.py

```diff
@@ -0,0 +1,93 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+from spectre import DataStructures
+import spectre.IO.H5 as spectre_h5
+from spectre import Informer
+import unittest
+import numpy as np
+import os
+import numpy.testing as npt
+
+
+class TestIOH5VolumeData(unittest.TestCase):
+    # Test Fixtures
+    def setUp(self):
+        # The First tests involve inserting vol files, the h5 file for
+        # these will be deleted and recreated for each test as needed.
+        self.file_name_w = "pythontest.h5"
+        # The other tests require external data, this file will not be changed
+        self.file_name_r = Informer.unit_test_path() + "IO/VolTestData.h5"
```

Comment:
I think `sys.path` has some sort of function that safely concatenates paths. We should use that :)

Reply:
@nilsdeppe I couldn't find anything in `sys.path` but it seems `os.path` has a function `join(base_dir, file_name)` which seems to be platform independent concatenation, is this what you meant?

Reply:
whoops, yes sorry :)

Reply:
No Problem!